### ISSUE TITLE
[Bug]: python doesn't work after installed on termux 

### ISSUE THREAD

**Issue by a-gamer** [termux/termux-app#3097]
### Problem description

I've  install python 3.10
 and try to use it .like python-V
termux  ask me to install  it over and over
![Screenshot_2022-11-16-07-56-00-053_com termux](https://user-images.githubusercontent.com/24799844/202096356-1a0c9c24-5ad8-4b18-8edf-9086672d24a0.jpg)
 see the sceen

### Steps to reproduce the behavior.

look to screenshots 

### System information

* Termux application version:
* Android OS version: 11
* Device model: poxo x3 nfc

**Comment by lybekk** [comment]
Open a new shell and see if it is available as the `python3.10` executable

**Comment by a-gamer** [comment]
yes it's  but most of tool use python direct  so i want call it python not python3.10

**Comment by a-gamer** [comment]
solved 
sudo update-alternatives --install /usr/bin/python3 python /usr/bin/python3.10 1
sudo update-alternatives --config python

**Comment by a-gamer** [comment]
yes it's install  in tur repo 
and  this is the command i used 
sudo update-alternatives --install /data/user/0/com.termux/files/usr/bin/python3 python /data/user/0/com.termux/files/usr/bin/python3.11 2
 it's  worked but when run any tools on termux  it's force upgrade to python3.11
so how solve this issue

**Comment by a-gamer** [comment]
also pip not working with this method
